kuma/users/checks.py

```diff
@@ -0,0 +1,18 @@
+import stripe
+from django.conf import settings
+from django.core.checks import Error, register
+
+
+@register()
+def stripe_check(app_configs, **kwargs):
+    if settings.STRIPE_SECRET_KEY == "testing" or (
+        not settings.STRIPE_SECRET_KEY and not settings.STRIPE_PLAN_ID
+    ):
```

Comment:
Also, why check if the `settings.STRIPE_SECRET_KEY` is not something? Why discriminate against `settings.STRIPE_PUBLIC_KEY`?

I think a MUCH simplified and straightforward logic would be this:
```python
if settings.STRIPE_PLAN_ID:
    try:
        stripe.Plan.retrieve(...)
    except ...
```

Because that way it *will* exercise and properly validate the value of `STRIPE_SECRET_KEY` and `STRIPE_PUBLIC_KEY` right?

Reply:
CI seems to run system checks, using the keys from `testing.py`. Also wasn't super happy with that check, but not sure how to do it better :/

Wrt the public key: it is not being tested atm. Your simplified version would only run the check if one of the keys is present but I think we agreed that we wanted to run it when either is present.

Reply:
1. Perhaps the CI runs the `migrate` command but that doesn't happen on an existing full mysql like the one I have inside my docker-compose. Still seems weird. Can you double check what would trigger it if you add a debug line of `if settings.STRIPE_PLAN_ID=='testing': raise Exception('why?!??!')`?

2. Yeah, perhaps test should be done if ANY of the `settings.STRIPE_*` things are truthy. After all, I did notice that the template code checks if the STRIPE_PUBLIC_KEY is truthy. So my initial suggestion would happily not catch any errors if `STRIPE_PLAN_ID== ''` but it would still try to use it in the template code because the `STRIPE_PUBLIC_KEY != ''`.

Reply:
>     1. Can you double check what would trigger it if you add a debug line of `if settings.STRIPE_PLAN_ID=='testing': raise Exception('why?!??!')`?

https://travis-ci.org/mdn/kuma/jobs/649393338

wrt 2) Good point, I extended the check to test for the presence of any of the keys

Reply:
`python manage.py merge`
What the heck does that mean?
I still don't get why something in `settings/testing.py` would get used outside of `pytest`. 

We can hack on this tomorrow in our 1:1.